packages/commands/README.md

```diff
@@ -1,6 +1,48 @@
 # Commands
 
-Commands is a generic package that allows registering and modifying commands to be displayed using the commands menu (Also called cmd+k).
+Commands is a generic package that allows registering and modifying commands to be displayed using the commands menu (also called `cmd+k`).
+
+## Types of commands
+
+There are two ways to register commands: static or dynamic. Both methods receive a command object as an argument, which provides:
```

Comment:
Could we introduce here a brief explanation of the difference between a static and a dynamic command? Can we provide any use cases as examples to choose one type or another?

Reply:
Good call out, I updated the description of Static Commands tp provide a bit more context.